CinemaTicketSalesSystem/CinemaTicketSalesBusinessLogic/Models/LearnMoreMovieModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CinemaTicketSalesBusinessLogic.Models
+{
+    public class LearnMoreMovieModel
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public DateTime DateOfPremiere { get; set; }
+        public string AgeRating { get; set; }
+        public virtual ICollection<GenresModel> Genres { get; set; }
+        public virtual ICollection<ProducersModel> Producers { get; set; }
```

Comment:
collections here should not be virtual
they are virtual id db models only for Entity Framework purposes.